entry.sh

```diff
@@ -1,7 +1,24 @@
 #!/usr/bin/env bash
 
+function stellar_core_init_db() {
+	if [ -f /.db-initialized ]; then
```

Comment:
The file should be in `/data` (that's the volume we use in the Dockerfile). Otherwise the file is not going to be there after recreating the container. And let's create a variable `DB_INITIALIZED=/data/.db-initialized` and use it here and in the `touch` command.